ycmd/completers/cpp/clang_completer.py

```diff
@@ -399,3 +425,64 @@ def _ResponseForLocation( location ):
                                       location.column_number_ )
 
 
+# Strips the following leading strings from the raw comment:
```

Comment:
Yeah. I understand there is some pythonic regex magic which allows for setting these out more neatly, but I wasn't sure I fully understood the subtlety, so went with a single line and a hefty comment